.travis.yml

```diff
@@ -37,68 +272,34 @@ addons:
     - python3-all-dev
     - libmpc-dev
 
-python:
-  - 3.6
-  - 2.7
-  - pypy
-
-
-# Multiple envs will cause multiple build across multiple python versions.
-# We test the following:
-# 1. PySMT w/o solvers
-# 2. PySMT with one solver at the time
-#
-env:
-  matrix:
-    - PYSMT_SOLVER="all"   PYSMT_GMPY="TRUE"
-    - PYSMT_SOLVER="all"   PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="None"  PYSMT_GMPY="FALSE"   PYSMT_CYTHON="TRUE"
-    - PYSMT_SOLVER="None"  PYSMT_GMPY="FALSE"   PYSMT_CYTHON="FALSE"
-    - PYSMT_SOLVER="msat"  PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="z3"    PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="cvc4"  PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="yices" PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="bdd"   PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="picosat" PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="btor"  PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="msat_wrap" PYSMT_GMPY="FALSE"
-    - PYSMT_SOLVER="z3_wrap"   PYSMT_GMPY="FALSE"
-
-matrix:
-  fast_finish: true
-
-  exclude:
-    - python: pypy
-      env: PYSMT_SOLVER="all"   PYSMT_GMPY="TRUE"
-    - python: pypy
-      env: PYSMT_SOLVER="all"   PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="None"  PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="msat"  PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="z3"    PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="cvc4"  PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="yices" PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="bdd"   PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="picosat" PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="btor"  PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="msat_wrap" PYSMT_GMPY="FALSE"
-    - python: pypy
-      env: PYSMT_SOLVER="z3_wrap"   PYSMT_GMPY="FALSE"
+before_install: |
```

Comment:
This should go into ci/travis_before_install.sh

Reply:
:+1: